pages/api/prescriptions/read.ts

```diff
@@ -0,0 +1,35 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
+import prisma from '../../../lib/prisma'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  // retrieve prescriptions by id, patientId, or don't specify body to retrieve all
+  if (req.method === 'POST') {
+    try {
+      const { id, patientId } = req.body
```

Comment:
nit: will this throw if there is no id to be destructed?  I think we should separate our routes to follow general software practices:
```
/id
/all
```

Reply:
since the else is a findMany, if theres no id specified all prescriptions are retrieved